server/@types/educationAndWorkPlanApi/index.d.ts

```diff
@@ -88,6 +82,12 @@ export interface components {
        * @example 1
        */
       sequenceNumber: number
+      /**
```

Comment:
I've re-imported the swagger spec from the API which makes the `stepReference` field optional